plugin/web/components/layout/WdContent.tsx

```diff
@@ -0,0 +1,21 @@
+import { Content } from "antd/lib/layout/layout";
+import { memo, ReactNode } from "react";
+
+import "../../../node_modules/antd/dist/antd.less";
+import MapSettingTab from "../Tabs/MapSettingTab";
+
+type Props = {
+  className?: string;
+  children?: ReactNode;
+  current?: string;
+};
+const WdContent: React.FC<Props> = ({ className, current, children }) => {
```

Comment:
What does Wd stand for? 
Also, This is inside the `web/components/layout` directory but has MapSettings specific content. Either make this more general for your needs and somewhere else make it more specific for map settings, or consider moving this into a different, map settings focused directory.
Since this is going to be a component for PLATEAU View this repo doesn't need to, and probably doesn't make sense to, follow Atomic design (atoms, molecules, organisms and pages). 
Try to plan out your directory structure and that'll help you keep organized.